lib/v2/china/maintainer.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     '/news/military': ['jiaaoMario'],
     '/news/:category?': ['jiaaoMario'],
+    '/finance/:category?': ['KingJem'],
```

Comment:
```suggestion
    '/finance/:category?': ['KingJem'],
    '/news/military': ['jiaaoMario'],
    '/news/:category?': ['jiaaoMario'],
```
Please insert new entries in alphabetical order.